support-models/src/main/scala/com/gu/support/workers/states/SendThankYouEmailState.scala

```diff
@@ -3,12 +3,15 @@ package com.gu.support.workers.states
 import java.util.UUID
 
 import com.gu.support.workers.{PaymentMethod, SalesforceContactRecord, User, _}
+import com.gu.support.encoding.CustomCodecs.{decodeLocalTime, encodeLocalTime}
+import org.joda.time.LocalDate
 
 case class SendThankYouEmailState(
   requestId: UUID,
   user: User,
   product: ProductType,
   paymentMethod: PaymentMethod,
+  firstDeliveryDate: Option[LocalDate],
```

Comment:
Will we not always have this field?

Reply:
This will always be None for Recurring Contributions and Digital Pack.